build/schedview/Dockerfile

```diff
@@ -4,16 +4,22 @@ ARG hub
 FROM ${hub}/scheduler:${cycle}
 
 ARG schedview
-ARG rubin_sim
 
 RUN git clone https://github.com/lsst/schedview.git
 
 WORKDIR /home/saluser/schedview
 
 RUN /home/saluser/.checkout_repo.sh ${schedview}
 
+ARG bokeh
 RUN source /home/saluser/.setup_sal_env.sh && \
-    mamba install -y bokeh pytest-flake8 pytest-black selenium firefox geckodriver build
+    mamba install -y bokeh=${bokeh} \
+    pytest-flake8 \
```

Comment:
Why are we installing packages for development into a deployment container?

Reply:
I agree.. If I remember correctly there was something in this package that require these additional packages.. But this entire thing needs to be reworked once schedview becomes a conda package. So, in the end we should be doing `mamba install schedview`, but we are not there yet..